File: tests/fixtures/openai-realtime/openai-realtime-verify-deprecated-session-fields-broken/01-caller-config.ts
const callerConfigMsg = {
  type: 'session.update',
  session: {
    modalities: ['text', 'audio'],
    instructions: 'translate from Spanish to English',
    input_audio_format: 'g711_ulaw',
    output_audio_format: 'g711_ulaw',
    turn_detection: { type: 'server_vad' },
    // Setting temperature to minimum allowed value to get deterministic translation results
    temperature: 0.6,
  },
};

export default callerConfigMsg;

File: tests/fixtures/openai-realtime/openai-realtime-verify-deprecated-session-fields-broken/01_caller_config.py
import json


async def configure_caller_session(ws):
    await ws.send(
        json.dumps(
            {
                "type": "session.update",
                "session": {
                    "instructions": "Translate caller audio to English.",
                    "temperature": 0.6,
                },
            }
        )
    )
